src/riak_kv_vnode.erl

```diff
@@ -110,6 +110,21 @@
                 reqid :: term(),
                 target :: pid()}).
 
+-record(counter_state, {
```

Comment:
Hey @macintux, I'm sorry, that wasn't cool on my part. This is at least the 3rd review, and I appreciate more eyes. It's late and I shouldn't have responded until morning. What I mean is, if you want to rename it, please be my guest, maybe it would help a future maintainer understand the purpose of the counter.